al_aws.js

```diff
@@ -216,6 +216,23 @@ var setEnv = function(vars, callback) {
     });
 };
 
+var uploadS3Object = function ({ data, key, bucketName }, callback) {
+    var s3 = new AWS.S3();
+    // Setting up S3 upload parameters
+    const parseData = typeof data !== 'string' ? JSON.stringify(data) : data;
+    let bucket = bucketName ? bucketName : process.env.aws_lambda_s3_bucket;
```

Comment:
we should be using different S3 bucket created for DLQ data specifically.

Reply:
created new bucket and updating as  dL_s3_bucket_name